lisa/operating_system.py

```diff
@@ -806,11 +806,14 @@ def _install_packages(
         add_args = self._process_extra_package_args(extra_args)
         command = (
             f"DEBIAN_FRONTEND=noninteractive apt-get {add_args} "
-            f"-y install {' '.join(packages)}"
+            f"-y --force-yes install {' '.join(packages)}"
```

Comment:
Will it bypass the version conflicts? Some version conflicts may raise errors, so we can know them. If it's force installed, it will be hidden.

Reply:
I've tested the changes in this file, still encounter below issue in 1/20 rate, need update and figure out effective way.
```
failed. AssertionError: [Failed to install ['iperf3'], please check the package name and repo are correct or not.
E: Unable to locate package iperf3

get unexpected exit code on cmd ['sudo', 'sh', '-c', 'DEBIAN_FRONTEND=noninteractive apt-get  -y --force-yes install iperf3 --allow-unauthenticated']] Expected <[0]> to contain item <100>, but did not
```

Reply:
The message says, "Unable to locate package iperf3". It shouldn't be related to the force parameter.